aws-android-sdk-appsync/src/main/java/com/amazonaws/mobileconnectors/appsync/subscription/mqtt/MqttSubscriptionClient.java

```diff
@@ -67,7 +67,7 @@ public void connect(final SubscriptionClientCallback callback) {
             mqttConnectOptions.setMqttVersion(MqttConnectOptions.MQTT_VERSION_3_1_1);
             mqttConnectOptions.setCleanSession(true);
             mqttConnectOptions.setAutomaticReconnect(true);
-            mqttConnectOptions.setKeepAliveInterval(5);
+            mqttConnectOptions.setKeepAliveInterval(60);
```

Comment:
As discussed 30 seconds.

Reply:
Will do.

Reply:
I believe it should be 15-20. We should check with @undefobj